lib/tasks/database_updates/backfill_publisher_selected_wallet.rake

```diff
@@ -0,0 +1,105 @@
+namespace :database_updates do
+  desc 'Backfill Publisher Selected Wallets'
+  task :backfill_publisher_selected_wallets => :environment do
+    def handle_gemini
+      now = Time.zone.now
```

Comment:
```suggestion
      now = Time.now
```

Reply:
Any reason to go against the recommendation [here](https://rails-bestpractices.com/posts/2014/10/22/use-time-zone-now-instead-of-time-now/)?

Reply:
This is now gone.